napari/components/_tests/test_add_layers.py

```diff
@@ -72,7 +72,7 @@ def test_viewer_open_no_plugin(tmp_path):
     viewer = ViewerModel()
     fname = tmp_path / 'gibberish.gbrsh'
     fname.touch()
-    with pytest.raises(ValueError, match='No plugin found capable of reading'):
+    with pytest.raises(ValueError):
```

Comment:
I prefer to validate the error message: At least to check if the exception contains file name. 
```suggestion
    with pytest.raises(ValueError, match=".*gibberish.gbrsh.*"):
```